ocsgw/src/main/java/com/telenordigital/ocsgw/OcsServer.java

```diff
@@ -40,15 +41,21 @@ public void init(Stack stack, AppConfig appConfig) {
 
         switch (appConfig.getDataStoreType()) {
             case DataSourceType.GRPC:
-                logger.info("Using GrpcDataSource");
+                LOG.info("Using GrpcDataSource");
                 source = new GrpcDataSource(appConfig.getGrpcServer(), appConfig.encryptGrpc());
                 break;
             case DataSourceType.LOCAL:
-                logger.info("Using LocalDataSource");
+                LOG.info("Using LocalDataSource");
                 source = new LocalDataSource();
                 break;
+            case DataSourceType.PROXY:
```

Comment:
As of now, PROXY is just a proxy for one type - GRPC.
In future, if we add new type - PUBSUB, then it should be configurable to choose secondary source for PROXY.
But, as of now, this is not worth.